src/util/Constants.js

```diff
@@ -19,6 +19,7 @@ const AllowedImageFormats = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
 const AllowedImageSizes = Array.from({ length: 9 }, (e, i) => 2 ** (i + 4));
 
 function makeImageUrl(root, { format = 'webp', size } = {}) {
+  if (typeof size !== 'number') throw new TypeError('INVALID_TYPE', 'size', 'number');
```

Comment:
Did anyone actually test this? 
Because just doing anything without the size field being there errors out. 
Ex: `.iconURL()`, `.avatarURL()`, `.displayAvatarURL()`

cc: @ImRodry @vladfrangu

Reply:
oh shit

Reply:
Addressed in #6686, guess I forgot to test not passing a size, sorry!